app/controllers/organizations/staff/dashboard_controller.rb

```diff
@@ -1,11 +1,58 @@
 class Organizations::Staff::DashboardController < Organizations::BaseController
+  before_action :context_authorize!, only: %i[index incomplete_tasks overdue_tasks]
+  include Pagy::Backend
   layout "dashboard"
 
   def index
     @user = current_user
     @organization = Current.organization
```

Comment:
And this @organization? :)

Reply:
This was already part of existing code so did not remove it and I see it is currently used in view https://github.com/rubyforgood/pet-rescue/blob/main/app/views/organizations/staff/dashboard/index.html.erb#L17

Reply:
Ah yes ok thank you.